src/controllers/userController.js

```diff
@@ -6,6 +9,12 @@ import strings from '../utils/stringsUtil';
 import hashPassword from '../utils/hashPassword';
 import generateToken from '../utils/generateToken';
 
+const { users } = models;
+
+dotenv.config();
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+const { APP_URL_BACKEND } = process.env;
```

Comment:
You should also make sure **APP_URL_BACKEND** is then heroku hostname when deployed.
And define all new env variables in **.env.example**